docs/content/users/quickstart.md

```diff
@@ -47,6 +47,36 @@ ddev launch
     ddev launch
     ```
 
+## CakePHP
+
+Use a new or existing Composer project, or clone a Git repository.
+
+The CakePHP project type can be used with any CakePHP project >= 3.x, but it has been fully tested with CakePHP 5.x. DDEV automatically creates the `.env` file with the database information, email transport configuration and a random salt. If `.env` file already exists, `.env.ddev` will be created so you can take any variable and put into your `.env` file.
+
+=== "Composer"
+
+    ```bash
+    mkdir my-cakephp-app
+    cd my-cakephp-app
+    ddev config --project-type=cakephp --docroot=webroot --create-docroot --php-version=8.3
+    ddev composer create-project --prefer-dist cakephp/app:~5.0 my-cakephp-app
```

Comment:
I note that this isn't correct, so this wasn't walked through, please make sure to walk through and check it for validity, thanks!

```suggestion
    ddev composer create --prefer-dist cakephp/app:~5.0
```

Reply:
Thank you! it was left before changing to ddev composer. fixed